### PR TITLE
VEN-1267 | Add berth info to Payment & Contract pages

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Update types of order statuses 
- Add berth info to payment and contract pages

### Closes :no_good_woman:
[VEN-1267](https://helsinkisolutionoffice.atlassian.net/browse/VEN-1267)

### Manual testing :construction_worker_man:
- After getting a message for payment or an offer you should be able to see the berth information on the payment & contract views

## Screenshots :camera_flash:
image

### CODE REVIEW

File: src/locales/en.json [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
+    "berth": "Berth",
     "cancelled": "Cancelled",
     "dinghy_place": "Dinghy place",
+    "drafted": "Drafted",
     "due_date": "Due date",
     "electricity": "Electricity",
     "error": "Error",
     "expired": "Expired",
     "gate": "Gate",
+    "harbor": "Harbour",
```
Comment (TatuArvela) [inline src/locales/en.json:13]: Where's this coming from? If we want to use "harbour", there's at least the line "Harbor map (PDF)" that also needs to be changed

**Review by TatuArvela — APPROVED** [review]
One comment, LGTM 👍

File: src/locales/en.json (hunk 1 above)
Comment (Nizar-Rahme) [reply src/locales/en.json:13]: I don't know why it is initially "harbour" and not "harbor" on the customer UI, I changed it here for the sake of consistency.
I'll change "Harbor map (PDF)" as well.

Comment (Nizar-Rahme) [reply src/locales/en.json:13]: "Fixed" the remaining translations but pushed to the other PR 😅   https://github.com/City-of-Helsinki/berth-reservations-ui/pull/390

**Pull request merged**